sklearn/feature_extraction/text.py

```diff
@@ -608,12 +608,17 @@ class CountVectorizer(BaseEstimator, VectorizerMixin):
         absolute counts.
         This parameter is ignored if vocabulary is not None.
 
-    max_features : int or None, default=None
-        If not None, build a vocabulary that only consider the top
-        max_features ordered by term frequency across the corpus.
+    max_features : float in range [0.0, 1.0] or int, default=1.0
+        Build a vocabulary that only considers the top max_features ordered
+        by term frequency across the corpus.
+        If float, the parameter represents a proportion of the features after
+        stop words removal, integer absolute counts.
```

Comment:
Grammar here is a bit hard to read. How about a new sentence: "If an integer, it represents absolute counts"?